docs/contributor/resources/01-kyma.md

```diff
@@ -81,6 +81,18 @@ spec:
 > **CAUTION:**
 > Module referencing using NamespacedName and FQDN (Fully Qualified Domain Name) has been deprecated.
 
+### **.spec.modules[].managed**
+
+The **managed** field determines whether Lifecycle Manager manages the module or not. By default it is set to `true`. By setting it to `false`, the module is opted out of management by Lifecycle Manager. This means that the module and all its related resources remain in the remote cluster in their current state at the time of unmanaging the module, but Lifecycle Manager does NOT reconcile those any longer.
+
+Once a module has been successfully unmanaged, the module's status in **.status.modules[].state** shows `Unmanaged`. Once the `Unmanaged` state shows, the module's entry can also be deleted entirely from **.spec.modules[]**.
```

Comment:
I think you need to explicitly mention that also by removing the module entry when unmanaged, the module resources on the remote cluster will not be removed